data/json/items/tool/misc.json

```
@@ -43,6 +43,31 @@
     "color": "yellow",
     "flags": [ "WATCH", "ALARMCLOCK" ]
   },
+  {
+    "type": "TOOL",
+    "id": "eac_butane_can",
+    "name": "can of butane",
```

```suggestion
    "name": { "str": "can of butane", "str_pl": "cans of butane" },
```
We've got a different string format making life easier for translators